server/events/markdown_renderer.go

```diff
@@ -33,8 +34,20 @@ var (
 	// maxUnwrappedLines is the maximum number of lines the Terraform output
 	// can be before we wrap it in an expandable template.
 	maxUnwrappedLines = 12
+
+	//go:embed templates/*
+	templatesFS embed.FS
+	templates   *template.Template
 )
 
+func init() {
```

Comment:
How can we rewrite this without an `init` function?

https://stackoverflow.com/a/56039373

Reply:
I can probably encapsulate it into a nameless function.

Reply:
That would be great! Thank you for all the work you have put in so far. We're really close to getting this merged in. 😁